extensions/kusto/src/KustoServer.ts

```diff
@@ -0,0 +1,174 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
nit: typically ts files should be camelCased

Reply:
Changed to kustoServer